src/_guides/language/language-tour.md

```diff
@@ -1127,20 +1127,24 @@ is sometimes referred to as _arrow_ syntax.
   there, but you can use a [conditional expression](#conditional-expressions).
 {{site.alert.end}}
 
-A function can have two types of parameters: _required_ and _optional_.
-The required parameters are listed first, followed by any optional parameters.
-Optional parameters can be _named_ or _positional_.
+### Parameters
+
+A function can have any number of *required positional* parameters. These may be
+followed either by *optional positional* parameters or by *optional named*
+parameters.  A function can have optional positional parameters or optional
+named parameters, but not both.
+{% comment %}
+NULLSAFE: Replace "optional named" above with just "named" and add:
+
+A named parameter can be either optional or `required`.
```

Comment:
How about we add this now? Something like this.

Named parameters are optional unless they're specifically marked as required.

Reply:
Sounds good.  I added that to the "Named parameters" section.